### PR TITLE
test: intacct export settings edge cases and utility functions

### PR DESCRIPTION
**Pull request by JustARatherRidiculouslyLongUsername**
image

## Clickup
this closes https://app.clickup.com/t/86cwh86c9

### CODE REVIEW

File: src/app/integrations/intacct/intacct-shared/intacct-export-settings/intacct-export-settings.component.spec.ts [hunk 1]
```diff
@@ -167,15 +172,328 @@ describe('IntacctExportSettingsComponent', () => {
...
+    });
+
+    describe('Destination Options Handling', () => {
+      beforeEach(() => {
+        fixture.detectChanges();
+      });
+
+      it('should handle option search for reimbursable expense payment type', fakeAsync(() => {
+        const searchEvent = {
+          searchTerm: 'test',
```
Comment (DhaaraniCIT) [inline src/app/integrations/intacct/intacct-shared/intacct-export-settings/intacct-export-settings.component.spec.ts:380]: can. move it to fixture

**Review by DhaaraniCIT — APPROVED** [review]

**Review by ashwin1111 — APPROVED** [review]
